src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -1,8 +1,12 @@
 package lotto.domain
 
 object LottoStore {
-    fun purchase(money: Int, lottoGenerator: LottoGenerator): Lottos {
-        val count = money / Lotto.LOTTO_PRICE
-        return Lottos(List(count) { lottoGenerator.generateLotto() })
+    fun purchase(totalMoney: Int, manualLottoNumbers: List<List<Int>>, lottoGenerator: LottoGenerator): Lottos {
```

Comment:
```suggestion
    fun purchase(totalMoney: Int, manualLottoNumbers: Lottos, lottoGenerator: LottoGenerator): Lottos {
```

스타일 차이도 있을 것 같은데요~ manualLottoNumbers에 List<List<Int>>보다 Lotts를 넘기는 건 어떨까해요. 제안드린 이유는 자동로또와 다르게 수동 로또는 이미 만들어졌다고 1차적으로 만들어졌다고 생각해서요.